nix/desktop/realm-node/generate-nix.sh

```diff
@@ -20,12 +21,21 @@ cat << EOF > $input
 ]
 EOF
 
+# Specify the package.json file containing the build dependencies to install
+cat << EOF > $supplement_input
+[
+  "node-pre-gyp"
+]
+EOF
```

Comment:
What's the need for this multi-line syntax?

Reply:
this file might change often and include more modules, I just don't want to keep changing syntax as we add and remove modules.

Reply:
fair